apps/frontend/src/pages/CreateCharity.tsx

```diff
@@ -0,0 +1,671 @@
+import { ChangeEvent, FormEventHandler, useState } from "react";
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Checkbox,
+  Container,
+  FormControl,
+  FormControlLabel,
+  Grid,
+  Paper,
+  TextField,
+  Typography,
+} from "@mui/material";
+import SearchBar from "../components/searchBar";
+import { colors } from "../styles/colors";
+import _axios_instance from "../_axios_instance";
+
+interface OrganizationObject {
+  name: string;
+  logoUrl: string;
+  donated: number;
+  receiving: boolean;
+}
+
+interface CharityObject {
+  name: string;
+  description: string;
+  startDate: string;
+  endDate: string;
+  imageUrl: string;
+  organizations: OrganizationObject[];
+}
+
```

Comment:
I expect these Image objects are also being used in the create listing page too 🤔 Perhaps we could move it more a central location and use it there. Just to make the code more DRY but not a necessity

Reply:
Actually, these are all new for creating charities(At least the image upload ones are), since I need to keep track of the order of images here to attach them to the appropriate organization, whereas with the regular create listing page it's not so important.